src/Menu/Menu.js

```diff
@@ -125,10 +125,6 @@ class Menu extends Component<DefaultProps, AllProps, void> {
       // $FlowFixMe
       menuList.style.width = `calc(100% + ${size})`;
     }
-
-    if (this.props.onEnter) {
-      this.props.onEnter(element);
```

Comment:
@th317erd  FYI - I extracted the popover transition to `Grow` for the purpose of my autocomplete.  Here is a gist with that work completed: https://gist.github.com/rosskevin/1bd998940f8e9decb85aa624e45ed2d9